### PR TITLE
Update types.rst

### PR DESCRIPTION
**Pull request by GalRogozinski**
Added missing fields to transaction documentation

### CODE REVIEW

File: docs/types.rst [hunk 1]
```diff
@@ -155,6 +155,10 @@ API will build them for you, as the result of various API methods.
 
 Each ``Transaction`` has the following attributes:
 
+-  ``legacy tag``: A short message attached to the transaction.
+-  ``attachment_timestamp``: Estimated epoch time of the attachment to the tangle.
+-  ``attachment_time_lower_bound``: The lowest possible epoch time of the attachment to the tangle.
+-  ``attachment_time_upper_bound``: The highest possible epoch time of the attachment to the tangle.
 -  ``address: Address``: The address associated with this transaction.
    Depending on the transaction's ``value``, this address may be a
    sender or a recipient.
```
Comment (todofixthis) [inline docs/types.rst]: Could we keep the list of attributes in alphabetical order?

File: docs/types.rst [hunk 2]
```diff
@@ -155,6 +155,10 @@ API will build them for you, as the result of various API methods.
 
 Each ``Transaction`` has the following attributes:
 
+-  ``legacy tag``: A short message attached to the transaction.
+-  ``attachment_timestamp``: Estimated epoch time of the attachment to the tangle.
+-  ``attachment_time_lower_bound``: The lowest possible epoch time of the attachment to the tangle.
+-  ``attachment_time_upper_bound``: The highest possible epoch time of the attachment to the tangle.
```
Comment (todofixthis) [inline docs/types.rst]: Colud we add a PEP-484 type to each attribute?

E.g., `attachment_time_upper_bound: int: ...`

File: docs/types.rst [hunk 3]
```diff
@@ -155,6 +155,10 @@ API will build them for you, as the result of various API methods.
 
 Each ``Transaction`` has the following attributes:
 
+-  ``legacy tag``: A short message attached to the transaction.
```
Comment (todofixthis) [inline docs/types.rst]: s/b `legacy_tag`

Also, could we include a little information about the difference between `legacy_tag` and `tag`?

File: docs/types.rst (hunk 2 above)
Comment (GalRogozinski) [reply docs/types.rst]: sure

File: docs/types.rst (hunk 1 above)
Comment (GalRogozinski) [reply docs/types.rst]: sure

**Pull request merged**